src/docs/patterns/account-details.mdx

```diff
@@ -0,0 +1,140 @@
+
+This approach is most effective when there are relatively few account-related actions. You should **NOT** use this approach if you are using a navigation rail or a primary navigation mechanism other than a drawer.
+
+<React.Fragment>
+    <Divider />
+</React.Fragment>
+
+# Via User Menu
+
+<ImageGrid
+    images={[DesktopUserMenu]}
+    regularWidth
+    caption={`The user menu is controlled by an Avatar button and displays useful user-related information and actions.`}
+/>
+
+Another way to present user information and actions is with the user menu. The user menu appears as an Avatar in the [App Bar](/patterns/appbar), usually displaying a user-selected profile picture. If your application does not support user-elected images, you can default to displaying the user's initials or a generic [person icon](/style/iconography?icon=Person&isMaterial=true).When clicked, the avatar opens a menu with the user-related content, such as links to the user profile / settings, log out functionality, etc. (see the [contents](/patterns/account-details#contents) section below). Further, the menu should also display basic information about the user when opened, such as their name and / or username.
+
+This approach is most effective if you have more than a few account-related actions, or if you want to present more detailed information about a user. The user menu is a good option if you want to avoid overloading the navigation drawer. This technique can also be used in conjunction with a navigation rail.
+
+On mobile, the user menu content should be presented in a [Bottom Sheet](/patterns/overlay#bottom-sheet) instead of a dropdown menu.
+
+<React.Fragment>
+    <ImageGrid
+        images={[MobileUserMenu]}
+        caption={`On mobile-sized screens, the menu contents should be presented in a bottom sheet.`}
+    />
+    <Divider />
+</React.Fragment>
+
+# Contents
+
+Whether you are using the drawer or a dedicated user menu component, the content should be very similar. Most account functions can be located on the account settings page, but you should provide access to the most common / frequent actions in the drawer or user menu.
+
+You should include:
+
+-   A link to the account settings page
+-   A Log Out button
+-   Links to terms of service / privacy policy / license information (if applicable)
+
+When using the user menu, you will likely have more room available if you wish to include other frequently accessed account functions, such as change password, contact support, etc.
+
+<React.Fragment>
+    <Divider />
+</React.Fragment>
+
+# Design Specifications
+
+## Mobile
+
+<ImageGrid images={[Specs1, Specs3, Specs4]} regularWidth />
+
+## Desktop and Tablet
+
+If certain dimensions are not specified, refer to the dimensions suggested in the mobile section above.
+
+<ImageGrid images={[Specs2]} regularWidth />
```

Comment:
Again, this is a bad example of a user menu. Take out the blue text and stripes.

Reply:
And I suggest putting the user menu in its context. It's fairly confusing when the user menu is used on its own.